package.json

```diff
@@ -3,7 +3,8 @@
   "description": "The core `url` packaged standalone for use with Browserify.",
   "version": "0.10.2",
   "dependencies": {
-    "punycode": "1.3.2"
+    "punycode": "1.3.2",
+    "querystring": "^0.2.0"
```

Comment:
Please follow existing dependency style. This is not a safe way to include dependencies in modules.